src/data/data.js

```diff
@@ -181,16 +181,16 @@ const pageData = {
           'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.',
         contactInfo: [
           {
-            label: 'Dirección',
+            label: 'Dirección:',
             texts: ['Carrera 52 No. 67A-15', 'Torre C, Piso 7, Ruta N', 'Medellín, Colombia'],
           },
           {
-            label: 'Teléfono',
-            texts: ['+57 123 456 7890', '+57 123 456 7890'],
+            label: 'Teléfono:',
+            texts: ['+57 4 604 5535' + '  +57 4 604 5535'],
```

Comment:
This needs to be an array of texts, so please keep the same structure and go through it in the JS. Do not concatenate the string using the `+` operator.